src/components/Form.js

```diff
@@ -108,6 +109,16 @@ class Form extends React.Component {
         return _.first(_.keys(hasStateErrors ? this.state.erorrs : this.props.formState.errorFields));
     }
 
+    setPosition(element, position) {
+        if (!element.props.inputID && element.props.children) {
+            _.forEach(element.props.children, (child) => {
+                this.setPosition(child, position);
+            });
+        } else {
+            this.childPosition[element.props.inputID] = position;
+        }
```

Comment:
If `element.props.inputID` is false, but `element.props.children` is also falsey, then you will enter  in the `else` which I think is wrong.

Reply:
I put some console logs, and it doesn't happen in the debit card form, but in other forms like opening Personal details it does happen:

`element.props.inputID` is `undefined` and `element.props.children` is also `undefined`